src/fragments/start/getting-started/next/api.mdx

```diff
@@ -200,12 +200,20 @@ query ListPosts {
 
 In this section you will create a way to list and create posts from the Next.js application. To do this, you will fetch & render the latest posts from the server as well as create a new post on the client.
 
-Open **pages/index.js** and replace it with the following code:
+Ensure you have the following libraries installed:
 
 ```sh
 npm install aws-amplify @aws-amplify/ui-react
 ```
```

Comment:
Could this be removed in favor of the previous page's section on installing these libraries?https://docs.amplify.aws/start/getting-started/setup/q/integration/next/#initialize-a-new-backend

Reply:
I'm not sure.  I feel like folks landing on this page directly (e.g. via Google) might need to know these, thus why it's stated "Ensure you have..."